src/main/resources/web/app/qs-form.ts

```diff
@@ -158,16 +163,6 @@ export class QsForm extends LitElement {
     private _searchDebounced = debounce(this._search, 300);
 
     private _handleInputChange = (e: Event) => {
-        const target = e.target as HTMLFormElement;
-        if (this._isInput(target)) {
-            if (target.value.length === 0) {
-                this._clearSearch();
-                return;
-            }
-            if (target.value.length < this.minChars) {
-                return;
-            }
-        }
         this._searchDebounced();
```

Comment:
I am not sure we can remove all this, because it means it wait for debounce before acting, if their is an issue, it needs to be fixed here.

Reply:
I think we might need to clear in both when too short, in the `_handleInputChange` and in the `_search`. We might also need to clear on line 168

Reply:
How about cancelling the call? I've tested this change locally, and it seems to work well.

Reply:
It's very subtile and those things are really hard to catch as they happen on really particular condition. See https://github.com/quarkusio/quarkusio.github.io/pull/1911/files